src/Phinx/Db/Action/ChangeColumn.php

```diff
@@ -55,7 +55,7 @@ public function __construct(Table $table, $columnName, Column $column)
      * @param mixed $options Additional options for the column
```

Comment:
I wonder if we can check more of those "mixed" ones and fix up the docblock to be more specific.

Reply:
Fixed a couple of these. We can probably resolve a number of the `mixed` values throughout the codebase, but I'd rather not bloat this PR more than it already is and treat it more in the spirit of "mechanical replacement" and then do targeted replacements of the remaining mixed values.